chef/data_bags/crowbar/migrate/network/011_add_ovs_attributes.rb

```diff
@@ -1,17 +1,33 @@
 def upgrade(ta, td, a, d)
-  a["networks"]["nova_floating"]["add_ovs_bridge"] = ta["networks"]["nova_floating"]["add_ovs_bridge"]
-  a["networks"]["nova_floating"]["bridge_name"] = ta["networks"]["nova_floating"]["bridge_name"]
-  a["networks"]["nova_fixed"]["add_ovs_bridge"] = ta["networks"]["nova_fixed"]["add_ovs_bridge"]
-  a["networks"]["nova_fixed"]["bridge_name"] = ta["networks"]["nova_fixed"]["bridge_name"]
+  unless a["networks"]["nova_floating"].has_key? "add_ovs_bridge"
+    a["networks"]["nova_floating"]["add_ovs_bridge"] = ta["networks"]["nova_floating"]["add_ovs_bridge"]
```

Comment:
Metrics/LineLength: Line is too long. [104/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)